Build.cmake

```diff
@@ -60,8 +61,8 @@ ELSEIF(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
    ENDIF()
 ELSEIF(MSVC)
    set(PLATFORM_LINK_LIBRIES dbghelp)
-   set(CMAKE_CXX_FLAGS_RELEASE "/MT")
-   set(CMAKE_CXX_FLAGS_DEBUG "/MTd")
+   # set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MD")
```

Comment:
Creating a static library may well have value. But linking it to the static runtime (which the /MT switches do) will make it unusable in most scenarios: It can then only be linked to an executable or DLL which uses the static runtime as well, which is rare. 
The reason is that it is dangerous at best in MSVC to mix DLL and static runtime or to use several modules (DLLs and Executables) all using the static runtime. Each independent module (DLL or EXE) linking the static runtime introduces its own heap and thus anything allocated dynamically within the module and destroyed outside will lead to heap corruption. 

Therefore all projects I know use the dynamic runtime which is the CMake default.